thin-replica-server/include/thin-replica-server/thin_replica_impl.hpp

```diff
@@ -702,11 +709,51 @@ class ThinReplicaImpl {
       LOG_ERROR(logger_, "Failed to read from root cert - popen() failed, error: " << strerror(errno));
       throw std::runtime_error("Failed to read from root cert - popen() failed!");
     }
+
+    delim = (use_unified_certs) ? "O = " : "OU = ";
```

Comment:
If you need to use this definition more than once in the same class, define a data member and use it.